e2e/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+  },
+});
```

Comment:
Is this needed?

Reply:
Yes the config file is needed. When we remove it cypress adds it right back in again when attempt to run the tests.
![Screen Shot 2022-11-27 at 2 07 21 PM](https://user-images.githubusercontent.com/50315144/204162249-7417195f-2355-408b-a757-a7073375f1f5.png)